packages/node_modules/@webex/plugin-meetings/src/meeting-info/meeting-info-v2.js

```diff
@@ -77,6 +77,59 @@ export default class MeetingInfoV2 {
     });
   }
 
+  /**
+     * Fetches meeting info from the server
+     * @param {String} conversationUrl one of many different types of destinations to look up info for
+     * @returns {Promise} returns a meeting info object
+     * @public
+     * @memberof MeetingInfo
+     */
+  async createAdocSpaceMeeting(conversationUrl) {
+    const getInvitees = (particpants = []) => {
+      const invitees = [];
+
+      if (particpants) {
+        particpants.forEach((participant) => {
+          invitees.push({
+            email: participant.emailAddress,
+            ciUserUuid: participant.entryUUID
+          });
+        });
+      }
+
+      return invitees;
+    };
+
+    return this.webex.internal.conversation.get(
+      {url: conversationUrl},
+      {includeParticipants: true, disableTransform: true}
+    )
+      .then((conversation) => {
+        const body = {
+          title: conversation.displayName,
+          spaceUrl: conversation.url,
+          keyUrl: conversation.encryptionKeyUrl,
+          kroUrl: conversation.kmsResourceObjectUrl,
+          invitees: getInvitees(conversation.participants?.items)
+        };
+
+        const uri = this.webex.meetings.preferedWebexSite ?
```

Comment:
this is a temp workaround, right? do we have a jira for doing this properly? Also when we don't have preferedWebexSite then uri is empty and we still call this.webex.request with empty url

Reply:
This is not temp as for all the adhoc meetings we need to use the preferred webex site , For other meetings there is a JIRA to use the webex meeting domain to select the url 

I will add an error object if preferred webex site it not found